app/src/androidTest/java/org/mozilla/fenix/syncintegration/SyncIntegrationTest.kt

```diff
@@ -113,30 +132,48 @@ class SyncIntegrationTest {
         }
     }
 
-    /* These tests will be running in the future
-    // once the test above runs successfully and
-    // the environment is stable
-
-    // History item Fenix -> Desktop
+    // Bookmark item Fenix -> Desktop
     @Test
     fun checkBookmarkFromDeviceTest() {
-        tapInToolBar()
-        typeInToolBar()
-        seeBookmark()
-        mDevice.pressBack()
-        signInFxSync()
+        val defaultWebPage = "example.com".toUri()!!
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(defaultWebPage) {
+        }.openThreeDotMenu {
+            verifyAddBookmarkButton()
+            clickAddBookmarkButton()
+        }
+        browserScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openTurnOnSyncMenu {
+            useEmailInsteadButton()
+            typeEmail()
+            tapOnContinueButton()
+            typePassword()
+            sleep(TestAssetHelper.waitingTimeShort)
+            tapOnSignIn()
+        }
     }
 
-    // Bookmark item Fenix -> Desktop
+    // History item Fenix -> Desktop
     @Test
     fun checkHistoryFromDeviceTest() {
-        tapInToolBar()
-        typeInToolBar()
-        sleep(TestAssetHelper.waitingTime)
-        mDevice.pressBack()
-        signInFxSync()
+        val defaultWebPage = "example.com".toUri()!!
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(defaultWebPage) {
+        }
```

Comment:
Should be able to I think use ```.openThreeDotMenu``` here without needing to use `browserScreen {}`

Reply:
@AaronMT you are right! removing that line. Thanks!